docs/Downloaders/SABnzbd/Basic-Setup.md

```diff
@@ -99,10 +103,12 @@ Covered and fully explained in [SABnzbd - Paths and Categories](/Downloaders/SAB
 ![!Switches: Post-processing](images/sabnzbd-switches-post-processing.png)
 
 1. If your hardware isn't up to snuff, including cpu and/or io performance, disabling Direct Unpack and/or enabling Pause Downloading During Post-Processing can help. Defaults are fine for most hardware though.
+1. This should be checked off if you have any sort of decent internet. as the amount of time spent to grab pars to use if its needed for verification/repair is trivial to the time that a repair might run to fail to realize it needs more pars to grab the next part then to try again.
+1. Personal choice if you want to enable this option, its usually an easy check and does provide benefit if job didn't have par2 files. Not every release has a par-set or sfv file. but generally speaking if were talking about scene releases things should have both but of course depends on how its posted/how the indexer is generating the nzb if it includes it or what. sfv do get used by others and well a basic crc32 checksum is better than not to know if the file is good. Parsing a sfv file and comparing crc32 checksum for a file takes very little resources. and sure you might say its a waste if your doing stuff with pars because the par checksum/check there would just handle it. but its just one of those options it doesn't do any harm to outweigh the value of scenarios that it could be used when a par-set is missing
```

Comment:
```suggestion
1. It is your choice if you want to enable this option. It's usually an easy check and does provide benefits if the job doesn't have par2 files, as not every release has a par-set or SFV file. Generally speaking, if we're talking about scene releases, things should have both but this depends on how it's posted and how the indexer is generating the nzb. SFV is commonly used and a basic crc32 checksum is better not knowing if the file is good. Parsing an SFV file and checking the files' integrities takes very little resources. This may seem redundant given that par's checks would also handle this, however, the ease with which the check is done makes the downside almost non-existent.